src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        int randomNumber = random.nextInt(100);
+        String randomColor = ColorSupplier.getRandomColor();
```

Comment:
The method 'getRandomColor' is being called statically. According to the checklist, static methods are discouraged. It would be better to create an instance of 'ColorSupplier' and call the method on that instance.